src/cpp/session/modules/connections/SessionConnections.cpp

```diff
@@ -74,7 +76,7 @@ SEXP rs_connectionOpened(SEXP connectionSEXP)
    if (!r::sexp::isNull(actionList))
    {
       std::vector<std::string> actionNames;
-      error = r::sexp::getNames(actionList, &actionNames);
+      r::sexp::getNames(actionList, &actionNames);
```

Comment:
Yes, that error turned out to be too noisy -- if we can't extract the list of names it's fine as it just means we won't have any actions. Arguably some feedback for the package developer might be in order but that probably doesn't belong in our server logs. :-) (Elsewhere, errors which are severe enough to render the connection unusable should indeed be logged).